stellar-dotnet-sdk/responses/TransactionResponse.cs

```diff
@@ -57,7 +57,7 @@ public class TransactionResponse : Response, IPagingToken
         public List<string> Signatures { get; private set; }
 
         [JsonProperty(PropertyName = "fee_bump_transaction")]
-        public FeeBumpTransaction FeeBumpTransaction { get; set; }
+        public FeeBumpTransaction FeeBumpTx { get; set; }

```

Comment:
I think we shoul d keep original name here to be consistent with prior name to prevent breaking change.